input/projects/why-join-wip.md

```diff
@@ -0,0 +1,51 @@
+# Why should my project join the .NET Foundation?
+
+The .NET Foundation offers administrative and technical services to open source projects to help maintainers govern and develop their projects.
+
+.NET Foundation services aim to increase project stability and maturity and don't replace developers on member projects. Day-to-day development remains in the hands of project maintainers and contributors.
+
+Project benefits include:
+
+## Project guidance and mentoring
+
+New projects joining us will get mentorship on how to best run an open source community along with access to developers currently seeking .NET projects. We work with project leads to help ensure the project grows into a vibrant and welcoming community. And, of course, we love to share best practices for managing a friendly, collaborative, and innovative open source workplace, wherever you may be.
+
+## IP and legal
+
+Legal can be tricky. We can help. Projects in our organization will get assistance to organize and manage their IP assignments as well as infrastructure for tracking common, simple, Contribution License Agreements across all .NET Foundation projects. We can also help ensure that your collaborators respect the open source licenses and trademarks of projects assigned to the .NET Foundation. Need advice on legal issues? We offer support there as well.
+
+## Marketing and communications
+
+The more people who know about .NET, the better for the ecosystem. We work with Amazon, JetBrains, Microsoft, Progress, and other industry leaders to improve your project's exposure within the community at large. Your success is our success - we'd love to see you achieve the accolades you deserve!
+
+## CLA management
+
+A Contributor License Agreement (CLA) ensures a project has the necessary rights to code contributions.
+
+Our automated CLA system has been working well for GitHub-based projects, with a very high rate of developers able to easily sign their Contribution License Agreements electronically. This service is offered with full automated notification to the project committers regarding the CLA status of a pull request author.
+
+## Secret management
+
+Get a secure vault to store secrets related to your project (social media credentials, website logins, etc.). Share credentials securely between trusted maintainers on your project, and enjoy the peace of mind that they're safely saved
```

Comment:
**QUESTION:** Could you expand on what technology is and how it's different from GH repository/organisation secrets?

Reply:
I'm guessing it is Azure Key Vault? IDK. More detail here would be good.